packages/client-twitter/src/post.ts

```diff
@@ -124,7 +119,7 @@ export class TwitterPostClient {
             const randomMinutes =
                 Math.floor(Math.random() * (maxMinutes - minMinutes + 1)) +
                 minMinutes;
-            const delay = randomMinutes * 60 * 1000;
+            const delay = 1 * 60 * 1000;
```

Comment:
seems like a regression here

Reply:
ah sorry, gotta add that back in, PR wasn't quite ready for review just yet